### ISSUE TITLE
Can't run iOS with global rnv (0.28.0-alpha.27)

### ISSUE THREAD

**Issue by sanderlooijenga** [pavjacko/renative#333]
**Describe the bug**
When running the following command; `rnv run -p ios -r`, it exits at;

```
$ rnv run -p ios -r

┌──────────────────────────────────────────────────────────────────────────────┐
│                                                                              │
│        ██████╗ ███████╗███╗   ██╗ █████╗ ████████╗██╗██╗   ██╗███████╗       │
│        ██╔══██╗██╔════╝████╗  ██║██╔══██╗╚══██╔══╝██║██║   ██║██╔════╝       │
│        ██████╔╝█████╗  ██╔██╗ ██║███████║   ██║   ██║██║   ██║█████╗         │
│        ██╔══██╗██╔══╝  ██║╚██╗██║██╔══██║   ██║   ██║╚██╗ ██╔╝██╔══╝         │
│        ██║  ██║███████╗██║ ╚████║██║  ██║   ██║   ██║ ╚████╔╝ ███████╗       │
│        ╚═╝  ╚═╝╚══════╝╚═╝  ╚═══╝╚═╝  ╚═╝   ╚═╝   ╚═╝  ╚═══╝  ╚══════╝       │
│                                                                              │
│        Version: 0.28.0-alpha.27                                              │
│        WARNING: this is a prerelease version.                                │
│        Use "npm install rnv" for stable one.                                 │
│        https://renative.org                                                  │
│        🚀 Firing up!...                                                      │
│        $ rnv run -p ios -r                                                   │
│        Start Time: 10/28/2019, 12:43:06 PM                                   │
│                                                                              │
└──────────────────────────────────────────────────────────────────────────────┘
…
$ rnv run -p ios -r - getConfigProp:ios:idSuffix:undefined
$ rnv run -p ios -r - _parseXcodeProject - Starting!
```

**To Reproduce**
1. See above

**Expected behavior**
iOS to run on an emulator.

**Screenshots or copy&paste**
See above.

**Desktop (please complete the following information):**
 - OS: osx
 - Node Version: v11.10.0
 - NPM Version: 6.7.0
 - RNV Version: 0.28.0-alpha.27

**Comment by mihaiblaga89** [comment]
fixed in 0.28.0-alpha.30. Thank you for your contribution, kind sir!

**Comment by sanderlooijenga** [comment]
Confirmed it working again, no thank you dear fellow sir!
